Comparing source compatibility of D:\IdeaProject\flink\apache-flink\flink-java\target\flink-java-1.10-SNAPSHOT.jar against D:\Program Files\JetBrains\IntelliJ IDEA 2020.2.3\plugins\maven\lib\maven3\conf\repository\org\apache\flink\flink-java\1.10.1\flink-java-1.10.1.jar
***  MODIFIED CLASS: PUBLIC org.apache.flink.api.java.ExecutionEnvironment  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW CONSTRUCTOR: PUBLIC(+) ExecutionEnvironment(org.apache.flink.configuration.Configuration, java.lang.ClassLoader)
